### PR TITLE
fix(GroupChatPanel): drop down was showing empty entries

### LINKED ISSUE #5955: In group chat panel, if nothing is found, the drop-down list is empty shown
### Description
Steps to reproduce the issue:
- Create a new **group chat** or use an existing one.
- Navigate to `more options` / `Add / remove members` (admin).
- Introduce chars to try to find a contact to add.  If nothing is found, the empty white list is shown under the group chat panel.
image

**NOTE:** See the behavior of the `StatusTagSelector` component. Maybe the issue is there.

Found by @elina2015 here: https://github.com/status-im/status-desktop/pull/5823#issuecomment-1143102137

### PR DESCRIPTION
**Pull request by alexandraB99**
Closes #5955

### What does the PR do
GroupChatPanel - drop down was showing empty entries 

### Affected areas
GroupChatPanel

### Screenshot of functionality
https://user-images.githubusercontent.com/31625338/172180050-27450fc9-1a63-42f7-aac0-9089387a54a9.mov

### CODE REVIEW

**Review by PascalPrecht — APPROVED** [review]

File: ui/app/AppLayouts/Chat/panels/GroupChatPanel.qml [hunk 1]
```diff
@@ -150,7 +156,7 @@ RowLayout {
          showSortedListOnlyWhenText: true
          toLabelText: qsTr("To: ")
          warningText: qsTr("USER LIMIT REACHED")
-         onTextChanged: sortModel(d.contactsModel)
+         onTextChanged: sortModel(d.buildContactsModel())
```
Comment (noeliaSD) [inline ui/app/AppLayouts/Chat/panels/GroupChatPanel.qml:159]: Why do we need here to rebuild again the `contactModel` list (this list should not change)?

**Review by noeliaSD — CHANGES_REQUESTED** [review]
2 comments:
- In latest master and in current spec of the group chat (in brief it will change but currently, until the admin is not displayed readonly), admin will not be displayed in the list of members of the group. I've been quickly reviewing the changes you made but I cannot see which is the difference that makes that now the admin is also shown in the group panel list.
- Please, see the behavior I'm getting when testing.. I'm expecting to see the dropdown with options but after removing the members in the list, the dropdown doesn't show the expected contacts to be added again:

https://user-images.githubusercontent.com/97019400/172574520-14cfc76e-e9f8-491b-9a9b-15af928fd882.mov

**Review by noeliaSD — APPROVED** [review]
Retested!

**Review by Hbouaz — APPROVED** [review]
Passed testing

**Pull request merged**